README.md

```diff
@@ -76,10 +83,23 @@ mkdir -p ~/geoserver_data
 docker run -d -v $HOME/geoserver_data:/opt/geoserver/data_dir kartoza/geserver
 ```
 
-You need to ensure the ``geoserver_data`` directory has sufficinet permissions
+You need to ensure the ``geoserver_data`` directory has sufficient permissions
 for the docker process to read / write it.
 
+## Setting Tomcat properties
 
+To set Tomcat properties such as maximum heap memory size, create a `setenv.sh` file such as:
+
+```shell
+JAVA_OPTS="$JAVA_OPTS -Xmx1536M -XX:MaxPermSize=756M"
+JAVA_OPTS="$JAVA_OPTS -Djava.awt.headless=true -XX:+UseConcMarkSweepGC -XX:+CMSClassUnloadingEnabled"
```

Comment:
One thing you may want to mention is creating a way to set the Geoserver data directory. 

http://docs.geoserver.org/stable/en/user/datadirectory/data-dir-setting.html#java-system-property

You may want to add that ability since it allows a user of this container to persist their Geoserver data.

Take a look at what I've done here:
- https://github.com/USGS-CIDA/DSASweb/blob/master/geoserver/dsas-geoserver/src/main/resources/docker/Dockerfile
- https://github.com/USGS-CIDA/DSASweb/blob/master/geoserver/dsas-geoserver/src/main/resources/docker/setenv.sh